packages/grafana-ui/src/components/VizTooltip/VizTooltipFooter.tsx

```diff
@@ -33,7 +33,7 @@ export const VizTooltipFooter = ({ dataLinks, annotate }: Props) => {
   return (
     <div className={styles.wrapper}>
       {dataLinks.length > 0 && <div className={styles.dataLinks}>{renderDataLinks()}</div>}
-      {annotate && (
+      {annotate != null && (
```

Comment:
this is a it faster than coercion from function to boolean. tooltips re-renders a lot, so worth it here.